manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -115,3 +115,18 @@ spec:
         apiserver_request_count{group="extensions",version="v1beta1",resource!~"ingresses|",client!~"hyperkube/.*|cluster-policy-controller/.*"}
       labels:
         severity: warning
+  - name: apiserver-requests-in-flight
```

Comment:
@smarterclayton are these approved now for telemetry?

Reply:
Yes, you have my approval for 4 series for this